packages/web/components/Guild/GuildLinks.tsx

```diff
@@ -1,44 +1,75 @@
-import { Button, Wrap, WrapItem } from '@metafam/ds';
+import { IconButton, Wrap, WrapItem } from '@metafam/ds';
 import { GuildFragmentFragment } from 'graphql/autogen/types';
 import React from 'react';
-import { FaDiscord, FaGlobe } from 'react-icons/fa';
+import { FaDiscord, FaGithub, FaGlobe, FaTwitter } from 'react-icons/fa';
+
+import { ProfileSection } from '../ProfileSection';
 
 type Props = {
   guild: GuildFragmentFragment;
 };
 
 export const GuildLinks: React.FC<Props> = ({ guild }) => {
   return (
-    <Wrap>
-      {guild.website_url ? (
-        <WrapItem>
-          <Button
-            as="a"
-            href={guild.website_url}
-            target="_blank"
-            size="xs"
-            colorScheme="blackAlpha"
-            leftIcon={<FaGlobe />}
-          >
-            Website
-          </Button>
-        </WrapItem>
-      ) : null}
-      {guild.discord_metadata?.inviteUrl ? (
-        <WrapItem>
-          <Button
-            as="a"
-            href={guild.discord_metadata.inviteUrl}
-            target="_blank"
-            size="xs"
-            bgColor="discord"
-            _hover={{ bgColor: 'discordDark' }}
-            leftIcon={<FaDiscord />}
-          >
-            Discord
-          </Button>
-        </WrapItem>
-      ) : null}
-    </Wrap>
+    <ProfileSection title="Links">
+      <Wrap>
+        {guild.website_url ? (
+          <WrapItem>
+            <a href={guild.website_url} target="_blank" rel="noreferrer">
+              <IconButton
+                variant="outline"
+                aria-label="Discord Server"
+                size="lg"
+                colorScheme="blackAlpha"
+                icon={<FaGlobe />}
+              />
+            </a>
+          </WrapItem>
+        ) : null}
+        {guild.discord_metadata?.inviteUrl ? (
+          <WrapItem>
+            <a
+              href={guild.discord_metadata.inviteUrl}
+              target="_blank"
+              rel="noreferrer"
+            >
+              <IconButton
+                variant="outline"
+                aria-label="Discord Server"
+                size="lg"
+                bgColor="discord"
+                icon={<FaDiscord />}
+              />
+            </a>
+          </WrapItem>
+        ) : null}
+        {guild.website_url ? (
+          <WrapItem>
+            <a href={guild.website_url} target="_blank" rel="noreferrer">
```

Comment:
wrong link. Let's not show this if a github URL is not available

Reply:
yeah I know, I put this in temporarily cause the fields aren't available yet

Reply:
OK, we don't want to show the github link is website_url is set though.